CHANGELOG.md

```diff
@@ -5,11 +5,16 @@
 ### Added
 - Add `vendor[1,2,3]-prompt` as options to ConnectionConfig-CT in `tcd_latest.xsd`.
 
+- Add JDBC Kerberos scenario for postgres 
+
 ### Changed
 - Validate length of `name` in Company-G in `connector_plugin_manifest_latest.xsd`
```

Comment:
This appears to be released as of 2020-09-21. [PR](https://github.com/tableau/connector-plugin-sdk/pull/605)

Reply:
Will remove this line